### PR TITLE
Connect redux toolkit with React 

### PR DESCRIPTION
**Pull request by shahzaman900**
### Connect the redux toolkit with React 

1. add store in app component 
2. use useSelect to get satte
3. use isedispatch to handle events
4. add book functionality
5. add removebook functionality

### CODE REVIEW

File: src/components/booklist.js [hunk 1]
```diff
@@ -1,29 +1,14 @@
...
+  const books = useSelector((state) => state.books);
   return (
     <>
       {
-        bookList.map((book) => (
-          <div key={book.Id}>
+        books.map((book) => (
+          <div key={book.item_id}>
             <Book book={book} />
           </div>
```
Comment (elisha2kyakpo1) [inline src/components/booklist.js:13]: - [ ] Kindly fix the errors from this component, and make sure the app runs without errors before submitting for a review.


react

**Review by elisha2kyakpo1 — APPROVED** [review]
Hi 👋 @sheezu900, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ The PR title and description are good and professional.
✔️ No linter errors.
✔️ Descriptive readme file👏

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/booklist.js [hunk 2]
```diff
@@ -1,29 +1,15 @@
...
-      auther: 'shah zaman',
-    },
-    {
-      Id: 3,
-      bookname: 'Angular Book',
-      auther: 'shah zaman',
-    },
-  ];
+  const books = useSelector((state) => state.books);
+  console.log(books);
```
Comment (BregornOriginal) [inline src/components/booklist.js]: - [ ] Kindly delete this console.log to clean your code and browser console.

File: src/components/booklist.js [hunk 3]
```diff
@@ -1,29 +1,15 @@
...
   return (
     <>
       {
-        bookList.map((book) => (
-          <div key={book.Id}>
+        books.map((book) => (
```
Comment (BregornOriginal) [inline src/components/booklist.js:10]: - [ ] Good work fixing the error in the browser! But yes, sometimes happens that you fix one error and there's a new one. That's progress! 
Kindly fix the new error in your browser console to make your console really clean 💯 

image

**Review by BregornOriginal — CHANGES_REQUESTED** [review]
Hi @sheezu900  

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

giphy (5)

## Highlights

✅ Linters are passing
✅ Good commit history
✅ Good work implementing changes requested by the previous reviewer

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TuyishimireEric — APPROVED** [review]
# Status: APPROVED 🟢  
### Hi  @sheezu900  👋 👋 

#### I'm @TuyishimireEric, Your code reviewer,
Good job on implementing the project requirements! 🥇 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it 🦾  
Congratulations! 🎉


## Highlight 👍 
✅ Made suggested changes

✅Used Redux store to display books

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/enus/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**